specs/interop/predeploys.md

```diff
@@ -245,7 +267,19 @@ function relayMessage(uint256 _destination, uint256 _source, uint256 _nonce, add
        _calldata: _message
     });
 
-    require(success);
+    if (success) {
+        successfulMessages[messageHash] = true;
+        delete failedMessages[messageHash];
+        emit RelayedMessage(messageHash);
+    } else {
+        // Will set the timestamp only once on the *first* failed execution to protect against extending
+        // the time-window and blocking the rollback.
```

Comment:
Is it ok if we move the inline code comments outside of the code block? Just to optimize the codeblock for less stuff going on and have the human readable text outside of it